File: DE_3/contestants/nguyen_the_khoi/SUMN.py
with open("SUMN.inp","r") as f:
    n=f.read().strip()
    f.close()
s=0
for i in range(len(n)):
    s+=int(n[i])
with open("SUMN.out","w") as f:
    f.write(str(s))
    f.close()

File: DE_3/contestants/nguyen_duy_hung/SUMN.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
    ifstream inp("SUMN.inp");
    ofstream out("SUMN.out");
    int n;
    inp >>n;
    int s=0;
    while(n>0)
    {
        s+=n%10;
        n/=10;
    }
    out << s;
    out.close();
    return 0;
}
